src/main/java/com/monumental/models/Monument.java

```diff
@@ -49,24 +46,35 @@
     @Column(name = "address")
     private String address;
 
+    @Column(name = "description")
+    private String description;
+
+    @Column(name = "inscription")
+    private String inscription;
+
     @ManyToMany(cascade = { CascadeType.ALL })
     @JoinTable(
             name = "monument_tag",
             joinColumns = { @JoinColumn(name = "monument_id", referencedColumnName = "id") },
             inverseJoinColumns = { @JoinColumn(name = "tag_id", referencedColumnName = "id") }
     )
-    private Set<Tag> tags;
+    private List<Tag> tags;
+
+    @OneToMany(mappedBy = "monument")
+    private List<Image> images;
 
     @OneToMany(mappedBy = "monument")
-    private Set<Image> images;
+    private List<Contribution> contributions;
+
+    @OneToMany(mappedBy = "monument")
+    private List<Reference> references;
 
     public Monument() {
 
     }
 
-    public Monument(String submittedBy, String artist, String title, Date date, String material, Double lat,
-                    Double lon, String city, String state) {
-        this.submittedBy = submittedBy;
+    public Monument(String artist, String title, Date date, String material, double lat,
+                    double lon, String city, String state) {
```

Comment:
One of the things I hate about java is the lack of dynamic constructors. In Salesforce's shitty version of Java you can do
```java
new Monument(artist="Bob", title="Some Monument");
```
and it's super convenient. To my knowledge there's no way to achieve that in Java other than accepting a list/set as a param which sucks because then it isn't typed/doesn't check that the fields exist

Reply:
This rant was brought to you by Sleep Deprevation®

Reply:
That's super nice.

Java also doesn't have freaking default parameters which is aids.